doc/nrf/releases_and_maturity/releases/release-notes-2.6.0.rst

```diff
@@ -0,0 +1,1416 @@
+  * Updated:
+
+    * Renamed nRF Device provisioning library to :ref:`lib_nrf_provisioning` library.
+    * The device mode callback sends an event when the provisioning state changes.
+    * The cbor code has been regenerated with zcbor 0.8.1 and adapted to API changes in zcbor 0.8.x.
+
+  * Fixed file descriptor handling by setting the :c:struct:`coap_client` structure's ``fd`` field to ``-1`` when closing the socket.
+
+* :ref:`lib_mqtt_helper` library:
+
+  * Added support for using a password when connecting to a broker.
+
+* :ref:`lib_lwm2m_client_utils` library:
+
+  * Updated:
+
+    * The Release Assistance Indication (RAI) support to follow socket state changes from LwM2M engine and modify RAI values based on the state.
+    * The library requests a periodic TAU (RPTAU) to match the client lifetime.
+
+* :ref:`lib_lwm2m_location_assistance` library:
+
+  * Updated the Ground Fix object to copy received coordinates to the LwM2M Location object.
+
+* :ref:`lib_fota_download` library:
+
+  * Added:
+
+    * The functions :c:func:`fota_download` and :c:func:`fota_download_any` that can accept a security tag list and security tag count as arguments instead of a single security tag.
+    * :c:enumerator:`FOTA_DOWNLOAD_ERROR_CAUSE_CONNECT_FAILED` as a potential error cause in  :c:enumerator:`FOTA_DOWNLOAD_EVT_ERROR` events.
+
+* :ref:`lib_nrf_cloud_rest` library:
+
+  * Updated the :c:struct:`nrf_cloud_rest_location_request` structure to accept a pointer to a :c:struct:`nrf_cloud_location_config` structure in place of the single ``disable_response`` flag.
+
+* :ref:`lib_wifi_credentials` library:
+
+  * Updated the PSA backend to use the PSA Internal Trusted Storage (ITS) for storing of Wi-Fi credentials instead of the Protected Storage.
+    This is changed because the PSA ITS is a better fit for storing assets like credentials.
+    When switching storage, the credentials need to be migrated manually, or the existing credentials are lost.
+  * Removed the ``CONFIG_WIFI_CREDENTIALS_BACKEND_PSA_UID_OFFSET`` Kconfig option.
+    Use the :kconfig:option:`CONFIG_WIFI_CREDENTIALS_BACKEND_PSA_OFFSET` Kconfig option instead.
+
+Libraries for NFC
+-----------------
+
+* Fixed an issue with handling zero size data (when receiving empty I-blocks from poller) in the :file:`platform_internal_thread` file.
+
+Security libraries
+------------------
+
+* Added the :ref:`trusted_storage_readme` library.
+
+* :ref:`nrf_security` library:
+
+  * Updated:
+
+    * The library no longer enables RSA keys by default, which reduces the code size by 30 kB for those that are not using RSA keys.
+      The RSA key size must be explicitly enabled to avoid breaking the configuration when using the RSA keys, for example by setting the Kconfig option :kconfig:option:`CONFIG_PSA_WANT_RSA_KEY_SIZE_2048` if 2048-bit RSA keys are required.
```

Comment:
```suggestion
      The RSA key size must be explicitly enabled to avoid breaking the configuration when using the RSA keys, for example, by setting the Kconfig option :kconfig:option:`CONFIG_PSA_WANT_RSA_KEY_SIZE_2048` if 2048-bit RSA keys are required.
```